subsys/nfc/Kconfig.nfc

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NFCT_IRQ_PRIORITY
+	int
+	prompt "Interrupt priority"
+	range 0 6
```

Comment:
Let's take into account suggestion from https://github.com/NordicPlayground/nrfxlib/pull/103, and modify the range accordingly.

Reply:
Ok added the similar limitations for range